examples/inverse/plot_mixed_norm_inverse_volume.py

```diff
@@ -0,0 +1,112 @@
+"""
+================================================================
+Compute sparse inverse solution with mixed norm: MxNE and irMxNE
+================================================================
+
+Runs (ir)MxNE (L1/L2 or L0.5/L2 mixed norm) inverse solver.
+L0.5/L2 is done with irMxNE which allows for sparser
+source estimates with less amplitude bias due to the non-convexity
+of the L0.5/L2 mixed norm penalty.
+
+See
+Gramfort A., Kowalski M. and Hamalainen, M.:
```

Comment:
These should probably be in NumPy docstring format, i.e. `... as in [1]_` above and this here:
```
References
----------
.. [1] blah blah blah
```